docs/mivot/features.rst

```diff
@@ -0,0 +1,8 @@
+********************************
```

Comment:
No need to create these empty, child namespace docs, add the automodapi line in the actual narrative docs. (For example look into e.g. docs/dal/index.rst). So, keep docs/mivot/index.rst and remove all the other rst files (unless index.rst becomes too long (~1000+ lines or with too many examples that take a long time to run))